apps/predictions/lib/repo.ex

```diff
@@ -13,6 +13,8 @@ defmodule Predictions.Repo do
   ]
 
   def all(opts) when is_list(opts) and opts != [] do
+    Logger.info("predictions_repo_all")
```

Comment:
I think for the purpose of simplifying the Splunk work, it still makes sense to use something like `prediction_repo_all_cache=call` `prediction_repo_all_cache=cache_miss`... so we are still querying for one `field` and then doing some calculation on it's value.  In my limited experience it can be tricky to compare the results of two distinctly different splunk queries